src/py21cmfast/inputs.py

```diff
@@ -1087,7 +1096,9 @@ class AstroParams(StructWithDefaults):
         "A_VCB": 1.0,
         "BETA_VCB": 1.8,
         "UPPER_STELLAR_TURNOVER_MASS": 11.447,  # 2.8e11
-        "UPPER_STELLAR_TURNOVER_INDEX": -0.61,
+        "UPPER_STELLAR_TURNOVER_INDEX": -0.6,
+        "SIGMA_LX": 0.5,
+        "CORR_LX": 0.2,  # NOTE (Jdavies): It's difficult to know what this should be, ASTRID doesn't have the xrays and I don't know which hydros do
```

Comment:
This note reminds me a good point: we should whenever possible note down where the default value comes from

Reply:
Thanks for adding the refs!